extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -36,7 +36,7 @@ public function index() {
 			'heroImageUrl' => $this->getHeroImageUrl(),
 			'inviteFriendsText' => $this->msg( 'communitypage-invite-friends' )->plain(),
 			'headerWelcomeMsg' => $this->msg( 'communitypage-tasks-header-welcome', $wgWikiTopic ?? $wgSitename )->parse(),
-			'adminWelcomeMsg' => $this->msg( 'communitypage-admin-welcome-message' )->text(),
+			'subheaderWelcomeMsg' => $this->msg( 'communitypage-subheader-welcome' )->text(),
```

Comment:
wait, it is: https://github.com/Wikia/app/blob/e86c5387e4b655ce0ee68d3e7fe5c25ffd3279e1/extensions/wikia/CommunityPage/CommunityPage.i18n.php#L30